lib/internal/Magento/Framework/View/Page/Config.php

```diff
@@ -339,6 +339,14 @@ public function getDescription()
         return $this->metadata['description'];
     }
 
+    /**
+     * @param string $title
+     */
+    public function setMetaTitle($title)
```

Comment:
Please add type hint to the method to have more strict interface. Also to be consistent with other methods in this class it would be good to add corresponding getter

Reply:
Do you mean the whole class or just this method?

Reply:
Uhm, but if I am going to refactor with strict types for the class, I risk to add a backward incompatiblity. Are you sure?